### PR TITLE
Update fastify range

### PR DESCRIPTION
**Pull request by cemremengu**
See #121. 

1.8.0 of stringify is introduced here https://github.com/fastify/fastify/commit/4741dc6e64ee8516cb0b40656bbae374fa823d7e#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,6 +23,6 @@ function fastifySwagger (fastify, opts, next) {
 }
 
 module.exports = fp(fastifySwagger, {
-  fastify: '>=0.39.0',
+  fastify: '>=1.12.1',
```
Comment (mcollina) [inline index.js]: I fear this might fail in v2, as it depends on fastify-static.

Comment (cemremengu) [reply index.js]: Should we bump `fastify-static` to the same version as well?

Comment (mcollina) [reply index.js]: no, not really. fastify-static should be `^1.0.0` ideally.

**Comment by cemremengu** [comment]
Closing this in favor of #130 and 1.x approach discussed

**Pull request closed without merging**